pkg/proxy/winuserspace/proxier.go

```diff
@@ -213,8 +213,7 @@ func (proxier *Proxier) addServicePortPortal(servicePortPortalName ServicePortPo
 			return nil, fmt.Errorf("could not parse ip '%q'", listenIP)
 		}
 		// add the IP address.  Node port binds to all interfaces.
-		args := proxier.netshIPv4AddressAddArgs(serviceIP)
-		if existed, err := proxier.netsh.EnsureIPAddress(args, serviceIP); err != nil {
+		if existed, err := proxier.netsh.AddIPAddress(serviceIP); err != nil {
```

Comment:
Why changing the name of this function? What it does is still the same: _ensuring_ the given IP address is assigned.

Reply:
Sure, I’ll change it back.